cmd/server/files.go

```diff
@@ -282,7 +282,12 @@ func deleteFile(logger log.Logger, repo ICLFileRepository) http.HandlerFunc {
 
 		w.Header().Set("Content-Type", "application/json; charset=utf-8")
 		w.WriteHeader(http.StatusOK)
-		json.NewEncoder(w).Encode(`{"error": null}`)
+
+		type response struct {
```

Comment:
Will you define this as a reusable type instead of duplicating it inside each method? Something like `type ErrorResponse struct`.